docs/api/plms.yaml

```diff
@@ -285,6 +285,926 @@ paths:
+          name: count
+          schema:
+            type: number
+          description: Return list of favorite books with given count.
+      tags:
+        - Books
+      responses:
+        200:
+          description: Successful Response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Books'
+        400:
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        401:
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        408:
+          description: Request Timeout
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        500:
+          description: Internal Error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+                /books/favorites:
+                  get:
+                    operationId: getFavoriteBooks
+                    summary: Favorite Books
+                    description: Get favorite books.
+                    parameters:
+                      - in: query
+                        name: count
+                        schema:
+                          type: number
+                        description: Return list of favorite books with given count.
+                    tags:
+                      - Books
+                    responses:
+                      200:
+                        description: Successful Response
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Books'
+                      400:
+                        description: Bad Request
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+                      401:
+                        description: Unauthorized
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+                      408:
+                        description: Request Timeout
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+                      500:
+                        description: Internal Error
+                        content:
+                          application/json:
+                            schema:
+                              $ref: '#/components/schemas/Error'
+  /books/not-placed:
```

Comment:
should we use a query parameter here?

Reply:
This is the situation where shelf_id is empty on books. Therefore, I was unsure whether to put up an URL or query. We can have a shelfId query; when shelfId=null is sent, the API can send not placed books. Which one do you prefer?

Reply:
We could use a query parameter instead of creating a new endpoint for retrieving different types of books